### PR TITLE
Updated examples on the homepage to be full width and higher res

### PR DESCRIPTION
**Pull request by angel**
After staring at the homepage for a bit, I decided that the button under the examples looked lonely when the example images were not full width as @KHarshawat had mentioned in this issue: https://github.com/18F/web-design-standards/issues/548, so I updated the examples to be higher res and full-width and shortened the images so it wouldn't take as much space on the page.

### CODE REVIEW

**Comment by maya** [comment]
@angel this has merge conflicts. After you fix, I can add this in.

**Comment by angel** [comment]
@maya fixed, let me know if I missed anything else.

File: pages/index.html [hunk 1]
```diff
@@ -152,26 +152,24 @@ <h3 class="usa-font-lead">A few samples of the web design standards at work:</h3
...
-    </div>
+    <h3 class="usa-example-heading usa-link-external">
+      <a class="usa-example-link" href="https://vote.usa.gov/">
+        Vote.usa.gov
+      </a>
+    </h3>
+    <img class="usa-img-example" src="{{ site.baseurl }}/assets-styleguide/img/home/example_voteUSAgov.jpg" alt="vote.usa.gov homepage">
+    <h3 class="usa-example-heading usa-link-external">
+      <a class="usa-example-link" href="{{ site.baseurl }}/assets-styleguide/img/home/example_VAappeals_full_mock.png">
+      Mockup of VA Appeals Modernization Screen
```
Comment (maya) [inline pages/index.html]: @angel can you indent this like the others?

File: pages/index.html [hunk 2]
```diff
@@ -152,26 +152,24 @@ <h3 class="usa-font-lead">A few samples of the web design standards at work:</h3
...
-      <h3 class="usa-example-heading usa-link-external">
-        <a class="usa-example-link" href="https://playbook.cio.gov">
-          U.S. Digital Service Playbook
-        </a>
-      </h3>
-      <img class="usa-img-example" src="{{ site.baseurl }}/assets-styleguide/img/home/example_playbook.png" alt="U.S. Digital Service Playbook homepage">
-    </div>
+    <h3 class="usa-example-heading usa-link-external">
+      <a class="usa-example-link" href="https://vote.usa.gov/">
+        Vote.usa.gov
```
Comment (maya) [inline pages/index.html]: @angel I think this should be lowercase. See @awfrancisco's edits to the home page.

**Comment by maya** [comment]
sparkly, shiny, beautiful :sparkles:

**Pull request merged**